client/src/components/Home.vue

```diff
@@ -21,6 +21,12 @@
     async mounted () {
       // TODO consider caching images
       this.results = await SpotifyService.getPlaylists()
+    },
+    methods: {
+      goToPlaylist: function (playlist) {
+        const newRoute = 'playlist/' + playlist.id
+        this.$router.push({path: newRoute})
```

Comment:
Can we use `this.$router.push({name: 'playlist', params: {playlist_id: playlist.id}})` here? I try to avoid hard coding any paths with vue-router

Reply:
I'll respond to all of the playlist_id-related comments here: yeah I am down to switch to the methods you're describing. All this is super hacked-together as I don't know loopback or vue :)